tools/hypestv/src/windows/hyperv.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! Functions for interacting with Hyper-V VMs.
```

Comment:
Perhaps this could be in a later PR, but it seems like it would be useful to split out stuff for interacting with hyper-v VMs into a separate crate so we could use it in petri for tests.